src/storage/src/control/generated/protos/rpc/build.rs

```diff
@@ -0,0 +1,39 @@
+// Copyright 2025 Google LLC
```

Comment:
Separately, we should figure out a way to generate these files to $TMP. We only use them when running `sidekick`.